### DISCUSSION TITLE
KAPE CLI comment

### DISCUSSION THREAD

**Discussion by godstoge** [EricZimmerman/KapeFiles#950] [Q&A] [answered]
Is there an official "comment" out character (#,//,!, etc) for the _kape.cli file? It would be handy being able to comment the individual CLI entries.

**Comment by AndrewRathbun** [comment, marked as answer]
```
# test comment
--tsource C: --tdest C:\temp\kapeclitest\tout --tflush --target KapeTriage --debug
```

I just tested this, and this worked ^^ however, the following did NOT work:

```
--tsource C: --tdest C:\temp\kapeclitest\tout --tflush --target KapeTriage --debug # test comment
```

So if you want to label individual entries, it seems like the best approach is to have the comment above your command, and label accordingly. Comments cannot be on the same line as the command(s) being run.

I did one last test:

```
# test comment
--tsource C: --tdest C:\temp\kapeclitest\tout --tflush --target KapeTriage --debug
# test comment
--tsource C: --tdest C:\temp\kapeclitest\tout --tflush --target KapeTriage --debug
# test comment
--tsource C: --tdest C:\temp\kapeclitest\tout --tflush --target KapeTriage --debug
# test comment
--tsource C: --tdest C:\temp\kapeclitest\tout --tflush --target KapeTriage --debug
```

and 4 instances of KAPE spawned and the commands started running successfully.
